Onboarding/TechicalOverview.md

```diff
@@ -0,0 +1,30 @@
+# CanDIG Technical Onboarding
+## CanDIG Project Overview
+The CanDIG project has a distributed architecture, meaning each site is meant to run their own _____(What do you call everything represented in the diagram together) and there is no centralized CanDIG server. Users can use their local CanDIG _____(^) to communicate with other sites. Every site is responsible for handling the authentication of it's users and the authorization of incoming requests from other sites. CanDIG uses an API driven approach, where users send requests to their CanDIG which communicate with other sites through ...?
+## Component Overview:
+### Tyk [Gateway, Pump]
+Tyk Gateway is an API managment platform that CanDIG uses to map API endpoints to endpoints in CanDIG server. When the user makes a request, it is made to Tyk, which forwards the request to the mapped endpoint in CanDIG server.  
+Tyk Pump collects metrics on the use of Gateway and has many methods for transmiting these metrics to various databases.
+### Keycloak
+Authentication. That's it.?
+### CanDIG server
+Fasciliates the communication between different sites running CanDIG. What else?
```

Comment:
Minor nit: spelling of Fasciliates. Meant Facilitates?

I think more can be said. We can perhaps look at what Version 2 is going to have in terms of microservices and note them down here as features of CanDIG server. cc @haoyuanli

Reply:
Sounds like a good idea

Reply:
sorry, i'm not exactly sure what i should prepare for this

Reply:
Nothing to prepare. What parts from current V1 monolith will lead to creation of services for V2, given V2 is a microservice based design. That is what I was referring to in my comment above. Just a way to figure out what candig-server does.